pyprof/prof/linear.py

```diff
@@ -107,7 +112,7 @@ def __init__(self, d):
         if any(x in d.name for x in Linear.gemmKernels):
             self.op_ = "linear"
         else:
-            assert any(x in d.name for x in Linear.biasKernels), f"Kernel data: {d}"
+            assert any(x in d.name for x in Linear.biasKernels), "Kernel Name: {}".format(d.name)
```

Comment:
Why change this back to .format syntax. According to this site, https://realpython.com/python-f-strings/, f-strings are a more recent pythonic way of programming string output.

Reply:
Fixed.